Docs/RuntimeAudit/08_緊密結合とレイヤー違反.md

```
@@ -85,9 +85,10 @@ private static bool TryResolve<T>(out T value, string context) where T : class
 
 ---
 
-## 【設計指摘】FindObjectsByType系によるシーン走査での結合（29箇所）
+## 【設計指摘】FindObjectsByType系によるシーン走査での結合（27箇所）
 
-シーン内オブジェクトの参照解決を実行時走査で行っている箇所が29ある。
+シーン内オブジェクトの参照解決を実行時走査で行っている箇所が27ある
+（全件は [付録B](#付録b-findobjectsbytype系-全27箇所) を参照）。
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)